docs/DeveloperGuide.md

```diff
@@ -114,7 +114,27 @@ The features of WerkIt! are split and grouped into 5 main features:
 5. [Search-related features](#search-related-features)
 
 ### Exercise-related features
-_to be updated_
+
+Format: `exercise <userAction> <keywords>`
```

Comment:
Line 118: if we agree to include Lines 134 to 137, this format can be kept! Else, can just remove `<keywords>`